### PR TITLE
Replacing localhost with ip address of backend

### LINKED ISSUE #48: API endpoint
# Description
The backend is now up and running 🚀
So lets hook frontend and backend up.

# Resolving this issue
- [x] Production build should point to the live API
- [x] `API-HOST` flag in `package.json`

The `API-HOST` flag would allow us to run the Frontend locally, without needing to start the Backend up

### PR DESCRIPTION
**Pull request by AriHrannar**
## Status
**Ready**

## Description
This is the elastic IP address I have created on AWS
NOTE: I might take down the server if I see my billing getting too high :)

Would be nice to not need to do a software "release" if the backend IP changes (I might change to Scaleway if the AWS costs go over $3 this month). That could be done by using docker and environment variables :) :) Might pick that up again!

Closes #48

### CODE REVIEW

**Comment by Ragnar-H** [comment]
@AriHrannar with the turn of the month being a few days ago.
Did you see any noticeable increase in your AWS price?
_and_
have you set up auto-shut down if the services cross a budget?

File: package.json [hunk 1]
```diff
@@ -56,5 +56,5 @@
   "lint-staged": {
     "*.js": "lint"
   },
-  "proxy": "http://localhost:8000"
+  "proxy": "http://54.93.203.184"
```
Comment (Ragnar-H) [inline package.json]: I believe the proxy should be used for DEV purposes.
[Documentation](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#proxying-api-requests-in-development)

Essentially the `proxy` property is expecting a setup where the frontend and backend live in the same domain, or at least API requests are directed to the same domain.

So if I'm not mistaken, this line should be removed from `package.json` and some endpoint logic should be added in the source files.
Preferably some logic where I could run something like `yarn run start:local` if I want it to run against my local instance of the backend

Comment (AriHrannar) [reply package.json]: Ill look into this

**Comment by AriHrannar** [comment]
I didnt have it turned on for a whole month, i got charged like $0.63 for August. Ill have it on for September and keep an eye on it. I have not setup any auto shutdown right now but might

**Comment by AriHrannar** [comment]
@Ragnar-H thoughts on this?

File: src/config.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const serverBaseUrl = process.env.NODE_ENV === 'production'
```
Comment (Ragnar-H) [inline src/config.js]: Good stuff 👍

Comment (Ragnar-H) [reply src/config.js]: An improvement would be to allow for overwriting the URL.

So let's say f.x. I've got the development app running, but I want it to talk to the remote backend (to get more data, or negate having to run docker on my machine, or proper testing of HTTP)

I'm imagining something like `yarn run start API_HOST='api-staging.veritus.io'`

Comment (skabbi) [reply src/config.js]: What remote backend would you be talking to except prod?

Comment (Ragnar-H) [reply src/config.js]: Staging

Which is non-existent right now I grant you

Comment (Ragnar-H) [reply src/config.js]: But when running the App in development mode, it's always pointing towards local host.
This is something I see myself changing quite often, where I don't want to go into the file and manually change it.

Instead I'd like to give the option via run commands

Comment (AriHrannar) [reply src/config.js]: Something about this [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-custom-environment-variables) that I can take a look at later.

**Pull request merged**